mobile/Components/DotTextInput.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState, useMemo } from 'react';
+import { TextInput } from 'react-native';
+import PropTypes from 'prop-types';
+
+function DotTextInput({ value, onChangeText, customStyle }) {
+  const [secure, setSecure] = useState(true);
+
+  const secureText = useMemo(() => {
```

Comment:
Not sure if this function is working properly right now since I am able to type the password in full view and only switches to dots once I click away